app/src/processing/app/contrib/AvailableContribution.java

```diff
@@ -348,7 +348,7 @@ public boolean writePropertiesFile(File propFile) {
         writer.println("lastUpdated=" + lastUpdated);
         writer.println("minRevision=" + minRev);
         writer.println("maxRevision=" + maxRev);
-        if (getType() == ContributionType.LIBRARY && importsList != null) {
+        if ((getType() == ContributionType.LIBRARY || getType() == ContributionType.MODE) && importsList != null) {
```

Comment:
Why is imports getting added to only library and modes, do example need imports ?